kustomize/overlays/prod/deployment.yaml

```diff
@@ -0,0 +1,11 @@
+- op: replace
+  path: /spec/template/spec/containers/0/image
+  value: docker-registry.default.svc:5000/canvas-app-explorer-test/canvas-app-explorer:latest
```

Comment:
Would be nice to parameterize this if we plan to run this on all releases.

Reply:
Or maybe it makes more sense to just update the value with the release tag on each release. I'm not sure.